nameservice/tutorial/21-build-run.md

```diff
@@ -1,143 +1,143 @@
----
```

Comment:
not sure why the diff is that big on this file?

Reply:
Whoa didn't notice that; no idea it looks exactly the same why is it showing as all changed?